src/components/Notification/Notification.tsx

```diff
@@ -395,6 +397,20 @@ const NotificationDropdown: React.FC = () => {
     }
   }
 
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (
+        dropdownRef.current &&
+        !dropdownRef.current.contains(event.target as Node)
```

Comment:
```suggestion
        dropdownRef.current &&
        !dropdownRef.current.contains(event.target as Node) &&
        !buttonRef.current?.contains(event.target as Node)
```
[P1] 버튼 내부 클릭시 동작하지 않도록 수정